app/models/school_validator.rb

```diff
@@ -8,11 +8,13 @@ class SchoolValidator
   attr_accessor :address_city,
                 :address_one,
                 :address_state,
+                :address_two,
```

Comment:
We need to include `attr_accessor` even for optional params now - although we still won't do any validations on them in the `...Validator` class.